holoviews/element/util.py

```diff
@@ -282,3 +282,21 @@ def connect_edges(graph):
         end = end_ds.array(end_ds.kdims[:2])
         paths.append(np.array([start[0], end[0]]))
     return paths
+
+
+def validate_regular_sampling(img, dimension, rtol=10e-9):
+    """
+    Validates regular sampling of Image elements ensuring that
+    coordinates the difference in sampling steps is at most rtol times
+    the smallest sampling step. By default ensures the largest
```

Comment:
I can't parse this docstring; "ensuring that coordinates the difference"?

Also, I think rtol isn't useful as a function argument if this utility is primarily called from hv's own code that doesn't supply that argument.  It could be a module-level variable or this could be a ParameterizedFunction, either of which would allow the tolerance to be overridden in cases where it isn't appropriate.